WebView/app/src/main/assets/main.js

```diff
@@ -25,7 +25,11 @@ function getData() {
 		var form = document.getElementById("location");
 		var currentLocation = form.options[form.selectedIndex].value;
 		document.getElementById("title").innerText = form.options[form.selectedIndex].text;
-        document.getElementById("description").innerText = data[currentLocation].description;
+        document.getElementById("currentTemp").innerText = data[currentLocation].currentTemp;
+        document.getElementById("shortDescription").innerText = data[currentLocation].description;
+        document.getElementById("longDescription").innerText = "Today in " + `${form.options[form.selectedIndex].text}`
+        + " there is a " + `${data[currentLocation].chancePrecip}` + " chance of precipitation and the humidity is "
```

Comment:
Please use a hanging indent for wrapped lines: https://google.github.io/styleguide/jsguide.html#formatting-indent

Reply:
Ok